reference/library/src/morpheus-master/sass/oxford/_overrides.scss

```diff
@@ -78,6 +78,11 @@ $calendar-event-background-color: darken(#F8F8F8, 45%);
     }
 }
 
+img {
```

Comment:
Maybe here if it applies across all tools and is to be contributed back:
reference\library\src\morpheus-master\sass\base\_extendables.scss

There is a description of the Sass directory structure here:
reference\library\src\morpheus-master\folder-structure.md